vic/drivers/shared_image/src/vic_init.c

```diff
@@ -986,8 +986,7 @@ vic_init(void)
                 }
                 sum += soil_con[i].AreaFract[j];
             }
-            // TBD: Need better check for equal to 1.
-            if (sum != 1.) {
+            if (!assert_close_double(sum, 1.0, 0., 0.001)) {
```

Comment:
Thanks @bartnijssen and @tbohn. I will update this PR to include a smaller threshold (probably 1e-10) and add clear warnings/documentation to that effect.